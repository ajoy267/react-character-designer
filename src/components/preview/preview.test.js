import { render } from '@testing-library/react';
import Preview from './preview';

test('testing if it renders Preview', () => {
  const container = render(<Preview />);
  expect(container).toMatchSnapshot();
});
