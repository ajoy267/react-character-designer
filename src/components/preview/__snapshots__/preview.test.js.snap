// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing if it renders Preview 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="render"
      >
        <div
          class="editor"
        >
          <div
            class="form-control"
          >
            <p>
              Head
            </p>
            <select>
              <option
                value="bird-head"
              >
                Bird
              </option>
              <option
                value="duck-head"
              >
                Duck
              </option>
              <option
                value="dog-head"
              >
                Dog
              </option>
              <option
                value="horse-head"
              >
                Horse
              </option>
            </select>
          </div>
          <div
            class="form-control"
          >
            <p>
              Middle
            </p>
            <select>
              <option
                value="blue-middle"
              >
                Blue
              </option>
              <option
                value="dress-middle"
              >
                Dress
              </option>
              <option
                value="pink-middle"
              >
                Pink
              </option>
              <option
                value="red-middle"
              >
                Red
              </option>
            </select>
          </div>
          <div
            class="form-control"
          >
            <p>
              Bottom
            </p>
            <select>
              <option
                value="blue-pants"
              >
                Blue Pants
              </option>
              <option
                value="dog-pants"
              >
                Dog Pants
              </option>
              <option
                value="leg-pants"
              >
                Leg Pants
              </option>
              <option
                value="white-pants"
              >
                White Pants
              </option>
            </select>
          </div>
        </div>
        <div
          class="catchphrase"
        >
          <div
            class="form-control"
          >
            <p>
              Add A Catch Phrase
            </p>
            <input
              style="height: 25px;"
              type="text"
              value=""
            />
          </div>
          <button>
            Save
          </button>
        </div>
        <div
          class="parts"
        >
          <img
            alt="bird-head"
            src="/assets/bird-head.png"
          />
          <img
            alt="blue-middle"
            src="/assets/blue-middle.png"
          />
          <img
            alt="blue-pants"
            src="/assets/blue-pants.png"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="render"
    >
      <div
        class="editor"
      >
        <div
          class="form-control"
        >
          <p>
            Head
          </p>
          <select>
            <option
              value="bird-head"
            >
              Bird
            </option>
            <option
              value="duck-head"
            >
              Duck
            </option>
            <option
              value="dog-head"
            >
              Dog
            </option>
            <option
              value="horse-head"
            >
              Horse
            </option>
          </select>
        </div>
        <div
          class="form-control"
        >
          <p>
            Middle
          </p>
          <select>
            <option
              value="blue-middle"
            >
              Blue
            </option>
            <option
              value="dress-middle"
            >
              Dress
            </option>
            <option
              value="pink-middle"
            >
              Pink
            </option>
            <option
              value="red-middle"
            >
              Red
            </option>
          </select>
        </div>
        <div
          class="form-control"
        >
          <p>
            Bottom
          </p>
          <select>
            <option
              value="blue-pants"
            >
              Blue Pants
            </option>
            <option
              value="dog-pants"
            >
              Dog Pants
            </option>
            <option
              value="leg-pants"
            >
              Leg Pants
            </option>
            <option
              value="white-pants"
            >
              White Pants
            </option>
          </select>
        </div>
      </div>
      <div
        class="catchphrase"
      >
        <div
          class="form-control"
        >
          <p>
            Add A Catch Phrase
          </p>
          <input
            style="height: 25px;"
            type="text"
            value=""
          />
        </div>
        <button>
          Save
        </button>
      </div>
      <div
        class="parts"
      >
        <img
          alt="bird-head"
          src="/assets/bird-head.png"
        />
        <img
          alt="blue-middle"
          src="/assets/blue-middle.png"
        />
        <img
          alt="blue-pants"
          src="/assets/blue-pants.png"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
